Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -183,9 +202,16 @@ class ActionOverlayTableViewHeader: UITableViewHeaderFooterView {
     }
 
     func configureWithSite(site: Site, image: UIImage?, imageBackgroundColor: UIColor?) {
-        self.siteImageView.backgroundColor = imageBackgroundColor
-        self.siteImageView.image = image?.createScaled(SimpleHighlightCellUX.IconSize) ?? SimpleHighlightCellUX.PlaceholderImage
-        self.titleLabel.text = site.title.characters.count <= 1 ? site.url : site.title
-        self.descriptionLabel.text = site.tileURL.baseDomain
+        siteImageView.backgroundColor = imageBackgroundColor
+
+        if AppConstants.MOZ_AS_PANEL {
```

Comment:
I had a hard time managing the favicon sizes of both the old TopSitesPanel and the new ASPanel at the same time. So, I used the AppConstant to resolve it for the time being. There's one side-effect. The favicons are not being restricted to a size in one approach. See these screenshots.
[History favicon when using TopSitesPanel](https://cloud.githubusercontent.com/assets/9949800/21687720/646556f2-d390-11e6-8e32-5011cd790288.png)
[History favicon when using ASPanel](https://cloud.githubusercontent.com/assets/9949800/21687729/692593dc-d390-11e6-8db8-5ca2f6801712.png)

I figured it wasn't worth spending time on that yet. But, releasing a bigger favicon in v7 and then a smaller favicon in v8 might not look elegant. So, do you have any advice on this?